pandas/tests/plotting/test_hist_method.py

```diff
@@ -238,6 +238,16 @@ def test_hist_layout(self):
         with tm.assertRaises(ValueError):
             df.hist(layout=(-1, -1))
 
+    @slow
+    # GH 9351
+    def test_tight_layout(self):
+        if self.mpl_ge_2_0_0:
+            df = DataFrame(randn(100, 2))
+            df.plot.hist()
+            self.plt.tight_layout()
```

Comment:
Would `_check_plot_works(df.hist); self.plt.tight_layout()` work? The method seems to be designed for testing parameter inputs, but `tight_layout` isn't a histogram parameter.